exporter/datadogexporter/metrics_exporter.go

```diff
@@ -226,7 +226,8 @@ func (exp *metricsExporter) PushMetricsData(ctx context.Context, md pmetric.Metr
 			exp.params.Logger.Debug("exporting native Datadog payload", zap.Any("metric", ms))
 			_, experr := exp.retrier.DoWithRetries(ctx, func(context.Context) error {
 				ctx = clientutil.GetRequestContext(ctx, string(exp.cfg.API.Key))
-				_, httpresp, merr := exp.metricsAPI.SubmitMetrics(ctx, datadogV2.MetricPayload{Series: ms})
+				param := datadogV2.NewSubmitMetricsOptionalParameters().WithContentEncoding(datadogV2.METRICCONTENTENCODING_GZIP)
```

Comment:
Can we make this const ?

Reply:
Why are you asking that? It would be quite an uncommon pattern.

Reply:
why create a variable for each request, that would be garbage collected .

Reply:
Good point, updated.